src/metabase/upload.clj

```diff
@@ -145,11 +146,24 @@
                 (t/plus (t/seconds 1))
                 (t/truncate-to :seconds))))))
 
+(def ^:private transliterator (Transliterator/getInstance "Any-Latin; Latin-ASCII"))
```

Comment:
We need a better one tbh, because this one generates weird translit. Results of this seem quite sane to me: `"Arabic-Latin; Bulgarian-Latin/BGN; Cyrillic-Latin/BGN; Greek-Latin/BGN; Any-Latin; Latin-ASCII"`.

Reply:
Can you give an example of what's weird about it?

Reply:
Would be good to have test cases driving it

Reply:
The main one is `"шо" => "sho"` (not "so"), but also `"хліб" => "khlib"` (rather than "hlib").

I also am struggling to find an example for Greek I figured out last time, my repl history is hiding that. :\ Also I think we can drop generic `Cyrillic-Latin/BGN` from that set, isn't really needed it seems.